atmat/lattice/atfastring.m

```diff
@@ -76,13 +76,13 @@
 nonlin_elem=atbaselem('NonLinear','DeltaQPass',...
     'Betax',lindata.beta(1),'Betay',lindata.beta(2),...
     'Alphax',lindata.alpha(1),'Alphay',lindata.alpha(2),...
-    'Qpx',xsi(1),'Qpy',xsi(2),...
+    'chromx_arr',[xsi(1)],'chromy_arr',[xsi(2)],...
     'A1',r1(1),'A2',r1(2),'A3',r1(4),...
     'T1',-orbit4{end},'T2',orbit4{end});
 nonlin_elemrad=atbaselem('NonLinear','DeltaQPass',...
     'Betax',lindata.beta(1),'Betay',lindata.beta(2),...
     'Alphax',lindata.alpha(1),'Alphay',lindata.alpha(2),...
-    'Qpx',xsi(1),'Qpy',xsi(2),...
+    'chromx_arr',[xsi(1)],'chromy_arr',[xsi(2)],...
```

Comment:
Same here: you should add chrom_maxorder=1 in the nonlinear element.

Reply:
done